### PR TITLE
Add error output to main.c

### PR DESCRIPTION
**Pull request by xVin**
targets #338

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -112,7 +132,7 @@ main(
      * LLLLL   OOO    OOO   P
      */
     ev_loop(default_loop, 0);
-
+    ws_log(&log_main, LOG_DEBUG, "Main loop exited normally.");
```
Comment (matthiasbeyer) [inline src/main.c]: Too much trailing whitespace here.

File: src/main.c [hunk 2]
```diff
@@ -122,7 +142,7 @@ main(
      * DDDD   EEEEE  I  N   N  I     T
      */
 cleanup:
-
+
```
Comment (matthiasbeyer) [inline src/main.c]: And here.

**Comment by matthiasbeyer** [comment]
ACK. @neithernut ?

**Comment by neithernut** [comment]
ACK.

**Pull request merged**